### DISCUSSION TITLE
Custom fields for tags

### DISCUSSION THREAD

**Discussion by PixedBo** [joomla/joomla-cms#42216] [Q&A]
Hello!
Is possible somehow to add custom fields for tags?
It would be great to extend each tag with more custom fields because now you can only set an image and a description.
Thanks

**Comment by Akshat111111** [comment]
We can implement it using - 
1)creating a custom Joomla plugin that adds the functionality . Joomla have a  plugins that can extend core functionality.
2)Another option is using  a custom fields extension like "FieldsAttach" to add extra custom fields to tags.
